### DISCUSSION TITLE
frigate 0.8.0 Beta 2

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#418] [Ask A Question]
I tried Beta 2 and noticed that the draw_zones is not working. Anyone else not able to see the zone line if enabled? same config does show the zone boundary line on beta 1 (I can't seem to copy/paste the config properly on GitHub using Windows 10)

`cameras:
  garage:
    ffmpeg:
      inputs:
        - path: rtmp://192.168.0.139:1935/bcs/channel0_ext.bcs?channel=0&stream=2&user=user&password=pass
          roles:
            #- detect
            - rtmp
            - clips
        - path: rtmp://192.168.0.139:1935/bcs/channel0_main.bcs?channel=0&stream=0&user=user&pass=pass
          roles:
            - detect
            - record
    height: 1536 #672
    width: 2048 #896
    fps: 5
    save_clips:
      enabled: True
      pre_capture: 5
      post_capture: 5
    record:
      enabled: true
    rtmp:
      enabled: True

    snapshots:
      show_timestamp: true
      draw_zones: True
      draw_bounding_boxes: False
      crop_to_region: True

    zones:
      front_yard:
        coordinates: 1,660,1,1532,2047,1535,2047,609,1616,403,1455,388,1282,408,1064,441,1106,472,227,625
`

**Comment by asemev** [comment]
Yes I also don't see the zone lines over the mjpg feed. But still detection works. 

But everything else works very well. My inference speed went down to around 14 which is awesome. And CPU loads also went down to around 15% for 2 cameras at 1080p.

My only issue is the snapshot attachments in notifications don't work... No attachment.

**Comment by blakeblackshear** [comment]
You can add several things dynamically. Add `zones=1` as a query string parameter. See the Readme for other options now.

**Reply by bm55** [reply]
Yes, the parameter does work. Thanks @blakeblackshear

**Comment by asemev** [comment]
Hi BM55,

Sorry for my ignorance, but how did you make the parameter work? I am total noob.
Also i read the README file inside the source.zip package in 0.8 beta2 but couldnt see any references to these parameters and how they can be used.
Could you please help?
Thx.
asemev

**Reply by asemev** [reply]
ah figured it out... http://192.168.178.84:5000/Garage?zones=1
it works :)
